files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -256,6 +256,9 @@
 [[ include "_labelingschema" . | indent 2 ]]
 [[ include "_node_role" . | indent 2 ]]
   metric_relabel_configs:
+  # drop unused rest client metrics
+  - source_labels: [__name__]
+    regex: rest_client_(rate_limiter_duration_seconds_bucket|request_size_bytes_bucket|response_size_bytes_bucket)
```

Comment:
No action, no replacement?
I guess this will replace with an empty name, but looks hard to read. Why not drop?

Reply:
I might have forgotten it 😅
I'll add it After lunch